internal/api/v1alpha1/authproxyworkload_types.go

```diff
@@ -176,6 +180,24 @@ type AuthProxyContainerSpec struct {
 	RolloutStrategy string `json:"rolloutStrategy,omitempty"`
 }
 
+type AdminServiceSpec struct {
```

Comment:
This is called the "Admin Server" in the proxy docs. I think we should try to match the proxy as much as possible to avoid introducing a new set of terms. See https://github.com/GoogleCloudPlatform/cloud-sql-proxy#localhost-admin-server.

Reply:
I'll update to call this "AdminServerSpec".